config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  mount_devise_token_auth_for 'User', at: 'auth'
```

Comment:
Not a correction, but keep in mind you could use both devise and devise-token-auth in the same model